src/ja/home.html

```diff
@@ -0,0 +1,106 @@
+---
+title: 'Open Web Advocacy'
+metaDesc: 'オープンなウェブの未来を擁護するために世界中から集まったソフトウェアエンジニアのグループ'
+permalink: '/ja/'
+layout: 'layouts/home.njk'
+isHome: true
+---
+
+<p class="home-intro">
+  私たちは<strong>オープンなウェブの未来</strong>を擁護するために世界中から集まったソフトウェアエンジニアのグループです。規制当局、立法者、政策立案者に対して複雑な技術の詳細情報を提供することで、この業界における競争を阻害する主要な問題とその解決方法の理解をサポートしています。
+</p>
+
+<div class="highlight">
+  <h2>危機にさらされているのは何か？</h2>
+
+  <div class="group">
+    <p>アプリケーション開発の未来すべてが危機にさらされています。規制や立法による変化なくしては、普遍的で、無料かつオープンな、一度書くだけでよく、どこにでもデプロイ可能な、アプリケーション頒布とデプロイのシステムを失うリスクがあります。このシステムは、企業と消費者のコストを大幅に削減するシステムです。
```

Comment:
```suggestion
    <p>アプリケーション開発の未来すべてが危機に瀕しています。規制や法律の変化が起こらなければ、企業と消費者の双方の負担を大幅に削減するアプリケーション公開の場が失われるおそれがあります。一度作ればどこにでも配布・導入できる、普遍的で、無料で、開かれたシステムが失われる可能性があるのです。
```
一文がすごく長くなってしまうのは避けたいが……